osi_common.proto

```diff
@@ -97,18 +97,24 @@ message Dimension3d
 // 
 // The preferred angular range is (-pi, pi]. The coordinate system is defined as right-handed. 
 //  
-// The rotations are to be performed \b roll \b first, \b pitch \b second and \b yaw \b third to follow the definition according to [1]. 
+// The rotations are to be performed \b yaw \b first, \b pitch \b second and \b roll \b third to follow the definition according to [1]. 
```

Comment:
Changing the order of rotations to yaw-pitch-roll makes this definition conform with various norms (DIN 8855 / DIN 70000 / DIN 9300). **HOWEVER**, this order of rotations applies to **INTRINSIC ROTATIONS**, i.e., every rotation is performed along the body axes. This means, that the statement in lines 102 and 103 **has to be removed!** All in all, I propose the following definition:

``The rotations are to be performed \b yaw \b first (around the z-axis), \b pitch \b second (around the new y-axis) and \b roll \b third (around the new x-axis) to follow the definition according to [1] (Tait-Bryan / Euler convention z-y'-x'').``

Explicitly stating the Tait-Bryan / Euler convention z-y'-x'' further reduces ambiguity and helps those who need to google this topic.
Note that the original definition of extrinsic rotations and the roll-pitch-yaw sequence **implicitly** produces the same result as the new definition of intrinsic rotations and the yaw-pitch-roll sequence. The major flaw of the old definition - apart from being non-standard - is, that the explanation of the extrinsic rotations is ambiguous: if the origin of the "reference coordinate frame" is chosen different from the body frame's origin, rotations would also result in translations!